lib/_http_client.js

```diff
@@ -70,7 +70,7 @@ function isInvalidPath(s) {
 }
 
 function validateHost(host, name) {
-  if (host != null && typeof host !== 'string') {
+  if (host !== null && typeof host !== 'string') {
```

Comment:
Changes like this are not safe because it changes behavior, specifically `undefined` will no longer be considered in addition to `null`.

Reply:
thank you for your suggest!